### ISSUE TITLE
Track posts on social media?

### ISSUE THREAD

**Issue by xaur** [RichardRed0x/decred-media-tracker#12]
If an article-looking content appears on Reddit or Facebook, or even a good thread on Twitter, shall we track it in this list?

Related to #6 (subjective judgement) and #11 (filtering junk), but this issue is specifically about refining inclusion criteria w.r.t. social media platforms.

**Comment by s-ben** [comment]
There are many free tools available to do this. Here's a recent'ish article reviewing the big ones,
https://www.brandwatch.com/blog/top-free-social-media-monitoring-tools/

I think this is one of those 'don't reinvent the wheel' scenarios, as there is so much out there already, and building and maintaining connections to ever-shifting APIs on Twitter, Facebook, Reddit, etc. will outpace our resources. The challenge will be not becoming dependent on a centralized service that may go down, become paid (the main biz model for free services), losing data privacy, etc. Might be worth looking for a well-supported open-source project doing this that we can integrate with.

**Comment by xaur** [comment]
Good idea to use existing tools. One of the key criteria for choosing which tool to invest into is the ability to export our data in a format we can deal with.

**Comment by s-ben** [comment]
That one is key for sure. I imagine many tools don't have ability to easily export data, as that would make the product less "sticky". We may end up having to work a little bit for the data whichever tool we choose.
